tests/cli/test_flow_run.py

```diff
@@ -563,3 +563,69 @@ async def test_when_num_logs_passed_with_reverse_param_and_num_logs(
             ],
             expected_line_count=10,
         )
+
+    async def test_passing_head_and_tail_raises(self, flow_run_factory):
+        # Given
+        flow_run = await flow_run_factory(num_logs=self.LOGS_DEFAULT_PAGE_SIZE + 1)
+
+        # When/Then
+        await run_sync_in_worker_thread(
+            invoke_and_assert,
+            command=[
+                "flow-run",
+                "logs",
+                str(flow_run.id),
+                "--tail",
+                "--num-logs",
+                "10",
+                "--head",
+            ],
+            expected_code=1,
+            expected_output_contains=(
+                "Head and Tail option can't be used together. \nPlease chosse only one!"
+            ),
+        )
+
+    async def test_default_tail_returns_default_num_logs(self, flow_run_factory):
+        # Given
+        flow_run = await flow_run_factory(num_logs=self.LOGS_DEFAULT_PAGE_SIZE + 1)
+
+        # When/Then
+        await run_sync_in_worker_thread(
+            invoke_and_assert,
+            command=["flow-run", "logs", str(flow_run.id), "-t"],
+            expected_code=0,
+            expected_output_contains=[
+                f"Flow run '{flow_run.name}' - Log {i} from flow_run {flow_run.id}."
+                for i in range(
+                    self.LOGS_DEFAULT_PAGE_SIZE, self.LOGS_DEFAULT_PAGE_SIZE - 20, -1
+                )
```

Comment:
```suggestion
                for i in range(
                    self.LOGS_DEFAULT_PAGE_SIZE - 9, self.LOGS_DEFAULT_PAGE_SIZE
                )
```

Since we aren't specifying the `reverse` flag (which reverses "the logs order to print the most recent logs first"), we would expect the logs to be printed TIMESTAMP ASC. I believe the above suggestion would test that.